Library/Homebrew/manpages/brew.1.md

```diff
@@ -55,14 +55,12 @@ With `--verbose` or `-v`, many commands print extra debugging information. Note
   * `cat` <formula>:
     Display the source to <formula>.
 
-  * `cleanup` [`--force`] [`--prune=`<days>] [`--dry-run`] [`-s`] [<formulae>]:
+  * `cleanup` [`--prune=`<days>] [`--dry-run`] [`-s`] [<formulae>]:
```

Comment:
No worries, that's what a second pair of eyes can be good for. :smiley_cat: Thanks acting quickly while I merely suggested I'd be addressing this soon in https://github.com/Homebrew/homebrew/issues/48139#issuecomment-172359201!